src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -26,17 +30,101 @@ export class AwsSamDebugConfigurationProvider implements vscode.DebugConfigurati
         token?: vscode.CancellationToken
     ): Promise<AwsSamDebuggerConfiguration | undefined> {
         if (!AWS_SAM_DEBUG_REQUEST_TYPES.has(debugConfiguration.request)) {
-            vscode.window.showInformationMessage('Invalid request type')
+            vscode.window.showErrorMessage('Invalid request type')
+
+            return undefined
+        }
+
+        if (!AWS_SAM_DEBUG_TARGET_TYPES.has(debugConfiguration.invokeTarget.target)) {
```

Comment:
I can't remember if invokeTarget is nullable. If so, you might need to do `debugConfiguration.invokeTarget?.target` to prevent a null access error on the target property.

Reply:
`invokeTarget` is required, and `target` is also required in the interface. That being said, that doesn't protect against someone straight up not putting it in their debug config; I'll play around with it and see.